src/servers/src/otlp/logs.rs

```diff
@@ -0,0 +1,516 @@
+    map.insert("ScopeVersion".to_string(), scope_version);
+    map.insert("ScopeAttributes".to_string(), scope_attrs);
+    map.insert("LogAttributes".to_string(), log_attrs);
+    PipelineValue::Map(Map { values: map })
+}
+
+fn build_identity_schema() -> Vec<ColumnSchema> {
+    [
+        (
+            "scope_name",
+            ColumnDataType::String,
+            SemanticType::Tag,
+            None,
+            None,
+        ),
+        (
+            "scope_version",
+            ColumnDataType::String,
+            SemanticType::Field,
+            None,
+            None,
+        ),
+        (
+            "scope_attributes",
+            ColumnDataType::Binary,
+            SemanticType::Field,
+            Some(ColumnDataTypeExtension {
+                type_ext: Some(TypeExt::JsonType(JsonTypeExtension::JsonBinary.into())),
+            }),
+            None,
+        ),
+        (
+            "resource_attributes",
+            ColumnDataType::Binary,
+            SemanticType::Field,
+            Some(ColumnDataTypeExtension {
+                type_ext: Some(TypeExt::JsonType(JsonTypeExtension::JsonBinary.into())),
+            }),
+            None,
+        ),
+        (
+            "log_attributes",
+            ColumnDataType::Binary,
+            SemanticType::Field,
+            Some(ColumnDataTypeExtension {
+                type_ext: Some(TypeExt::JsonType(JsonTypeExtension::JsonBinary.into())),
+            }),
+            None,
+        ),
+        (
+            "timestamp",
+            ColumnDataType::TimestampNanosecond,
+            SemanticType::Timestamp,
+            None,
+            None,
+        ),
+        (
+            "observed_timestamp",
+            ColumnDataType::TimestampNanosecond,
+            SemanticType::Field,
+            None,
+            None,
+        ),
+        (
+            "trace_id",
+            ColumnDataType::String,
+            SemanticType::Tag,
+            None,
+            None,
+        ),
+        (
+            "span_id",
+            ColumnDataType::String,
+            SemanticType::Tag,
+            None,
+            None,
+        ),
+        (
+            "trace_flags",
+            ColumnDataType::Uint32,
+            SemanticType::Field,
+            None,
+            None,
+        ),
+        (
+            "severity_text",
+            ColumnDataType::String,
+            SemanticType::Field,
+            None,
+            None,
+        ),
+        (
+            "severity_number",
+            ColumnDataType::Int32,
+            SemanticType::Field,
+            None,
+            None,
+        ),
+        (
+            "body",
+            ColumnDataType::String,
+            SemanticType::Field,
+            None,
+            Some(ColumnOptions {
+                options: StdHashMap::from([(
+                    "fulltext".to_string(),
+                    r#"{"enable":true}"#.to_string(),
+                )]),
+            }),
+        ),
```

Comment:
If I recall correctly there is a performance cost for opening full-text index. Should we add a config or parameter for controlling this behavior?

Reply:
We can turn it off by default and change it via alter option.